### DISCUSSION TITLE
I Love IT!

### DISCUSSION THREAD

**Discussion by aarmn** [4JX/L5P-Keyboard-RGB#36] [General]
after motherboard replacement because of device fault, the keys didnt match together and windows and vantage showed INVALID in my device section and refused to work, also I couldn't find any document anywhere, tried iCue official app and now, not only I have it working on windows but also on linux, thanks alot! also I would appreciate it if you tell me where you found docs to it?

**Comment by 4JX** [comment]
> I would appreciate it if you tell me where you found docs to it?

You mean the "how-to" on controlling the lights? That was done by another user (see bottom of the readme). I essentially just built upon it. The reverse engineering part can be done with Wireshark + USBPcap or whatever you find convenient.
